addons/stock_delivery/models/delivery_carrier.py

```diff
@@ -244,6 +244,8 @@ def fixed_send_shipping(self, pickings):
         return res
 
     def fixed_get_tracking_link(self, picking):
+        if self.tracking_url and picking.carrier_tracking_ref:
+            return self.tracking_url.replace("shippingNumber", picking.carrier_tracking_ref)
```

Comment:
Seeing what the `tracking_url` help says and the placeholder you use, shouldn't be `<shippingNumber>` instead of simply `shippingNumber` ?

Reply:
done